### ISSUE TITLE
Intermittent Bigtable acceptance test failure

### ISSUE THREAD

**Issue by blowmage** [GoogleCloudPlatform/google-cloud-ruby#2215]
We have seen occasional Bigtable acceptance test failures on `acceptance/bigtable/table/row_filter_test.rb:110`, both [on CI](https://circleci.com/gh/GoogleCloudPlatform/google-cloud-ruby/4366) and locally.

```
Finished in 49.641794s, 1.1482 runs/s, 5.9224 assertions/s.

  1) Failure:
DataClient Read Rows Filters::bigtable#test_0009_cells per column filter [/home/circleci/GoogleCloudPlatform/google-cloud-ruby/google-cloud-bigtable/acceptance/bigtable/table/row_filter_test.rb:110]:
Expected: 1
  Actual: 2
```

**Comment by blowmage** [comment]
@jiren can you take this one as well?

**Comment by blowmage** [comment]
I can't. I don't have permissions. You need to assign it to yourself.

**Comment by sduskis** [comment]
@jiren and @blowmage, has this been fixed?

**Comment by blowmage** [comment]
No. In fact, I opened #2257 last night to skip these tests to try to get our build passing again.

**Comment by blowmage** [comment]
This looks to be closed by #2281.
